lib/docs/scrapers/moment_timezone.rb

```diff
@@ -0,0 +1,30 @@
+module Docs
+  class MomentTimezone < UrlScraper
+    self.name = 'Moment.js Timezone'
+    self.slug = 'moment_timezone'
+    self.type = 'moment_timezone'
+    self.release = '0.5.37'
+    self.base_url = 'https://momentjs.com/timezone'
+    self.root_path = '/docs/'
+    self.initial_paths = %w(/docs/)
+    self.links = {
+      home: 'http://momentjs.com/timezone/',
+      code: 'https://github.com/moment/moment-timezone/'
+    }
+
+    html_filters.push 'moment_timezone/clean_html', 'moment_timezone/entries', 'title'
```

Comment:
```suggestion
    html_filters.push 'moment/clean_html', 'moment_timezone/entries', 'title'
```

Please reuse `moment/clean_html` (since it has the same definition)